contracts/modules/cover/Cover.sol

```diff
@@ -682,6 +682,25 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard, Mu
     stakingPoolFactory.changeOperator(_operator);
   }
 
+  function updateStakingPoolsRewardShares(
```

Comment:
Should this be AB members only?

Reply:
I did intend to restrict it but forgot. Was also initially thinking it could be safe enough to leave as is but on a second thought it's probably not the case. I want to restrict it but probably would use swap operator address to be able to run it ourselves.